backend/spvm.pm

```diff
@@ -0,0 +1,112 @@
+# Copyright © 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package backend::spvm;
+use strict;
+use base ('backend::virt');
+use testapi qw(get_var get_required_var check_var);
+
+use IO::Select;
+
+# supporting the minimal command set of NovaLink through a ssh tunnel
+
+sub new {
+    my $class = shift;
+    my $self  = $class->SUPER::new;
+    get_required_var('WORKER_HOSTNAME');
+
+    return $self;
+}
+
+# only define the novalink console - we leave the actual
+# poweron to the test
+sub do_start_vm {
+    my ($self) = @_;
+
+    # truncate the serial file
+    open(my $sf, '>', $self->{serialfile});
+    close($sf);
+
+    my $ssh = $testapi::distri->add_console(
+        'novalink-ssh',
+        'ssh-xterm',
+        {
+            hostname => get_required_var('NOVALINK_HOSTNAME'),
+            password => get_required_var('NOVALINK_PASSWORD'),
+            username => get_var('NOVALINK_USERNAME', 'root')});
+    $ssh->backend($self);
+
+    # remove backend.crashed
+    $self->unlink_crash_file;
```

Comment:
Test on my lab failed with unfound `unlink_crash_file`, I searched it but was not able found it...

Reply:
that function is dead by now - the commits have been lying around for a while

Reply:
So shall I remove that line?

Reply:
yes, see #930